tools/cli/air/src/trace/run/near.rs

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
I am curious if this module should be behind a feature flag near ?

Reply:
It is feature-flagged in the parent module.